webapp/src/main/webapp/src/app/sender/archive/sender-archive.component.html

```diff
@@ -0,0 +1,87 @@
+<!--View sync message overall status-->
+<div>
+	<div class="row">
+		<div class="col-md-6">
+
+		</div>
+
+		<div class="col-md-6">
+
+			<div>
+				<table style="left: 0;">
+					<tr>
+						<td></td>
+						<td colspan="4"></td>
+						<td colspan="2" i18n="@@common-start">Start Date</td>
+						<td></td>
+						<td i18n="@@common-end">End Date </td>
+						<td> </td>
+					</tr>
+
+					<tr>
+						<td>
+							<button class="btn btn-secondary" (click)="searchByPeriod($event)"> <span
+									i18n="@@common-search-period">Search by Period</span></button>
+						</td>
+						<td colspan="4"></td>
+						<td colspan="2">
+							<input type="date" [(ngModel)]="searchEvent.startDate" />
+						</td>
+						<td></td>
+						<td>
+							<input type="date" [(ngModel)]="searchEvent.endDate" />
+						</td>
+
+					</tr>
+
+				</table>
+			</div>
+		</div>
+	</div>
+
```

Comment:
Please always try to keep your code well formatted, why do you have this many extra blank lines here?

Reply:
DOne